.github/workflows/publish.yml

```diff
@@ -29,8 +29,8 @@ jobs:
         path: dist/
 
   publish-to-pypi:
-    name: >-
-      Publish to PyPI
+    name: Publish to PyPI
+    # TODO we need to make sure the tag matches the version!
     if: startsWith(github.ref, 'refs/tags/')  # only publish to PyPI on tag pushes
```

Comment:
This should be:

`if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')` to avoid pushing during a PR

Reply:
Well, that's not really a problem (?) I think this part is all good.